test.sh

```diff
@@ -1,4 +1,21 @@
 #!/usr/bin/env bash
-set -e
+set -ex
 
-./_integration-test/run.sh
\ No newline at end of file
+echo "Testing initial install"
+./install.sh
```

Comment:
Yeah, this is a no go for the getsentry/sentry repo. This file should be a new file named like `test-gha.sh` or something to avoid breakage from the other side.

Reply:
Got it, solved in [`004d41b` (#1479)](https://github.com/getsentry/self-hosted/pull/1479/commits/004d41bc4d64ff3af7eb970474c04ccb1a703b86)